tests/demoqa.spec.ts

```diff
@@ -43,4 +43,15 @@ test('test Radio Button', async ({ page }) => {
 	await page.locator('label[class^="custom-control"][for="impressiveRadio"]').click()
 	await expect(page.locator('//p[@class="mt-3"]').textContent()).resolves.toBe("You have selected Impressive");
 	// await page.locator('label[class^="custom-control"][for="noRadio"]').click() тут я бы завела баг, ибо кнопка не кликабельна
-})
\ No newline at end of file
+})
+
+test('test Web Tables', async ({ page }) => {
+	await page.locator('//span[text()="Web Tables"]').click()
+	await page.locator('#addNewRecordButton').click()
+	await page.locator('#firstName').fill('Margarita')
+	await page.locator('#lastName').fill('Artsiukhouskaya')
+	await page.locator('#userEmail').fill('lubanecmargo@yandex.ru')
+	await page.locator('#age').fill('30')
+   await page.locator('#salary-label').fill('1000')
+   await page.locator('#department').fill('QA')
+}
```

Comment:
Let's add a check that a new record has been added to the table and the record contains the values ​​​​that you entered

Reply:
You also can put all the data into variables so that when checking that the record was added correctly, you don’t duplicate the code